### PR TITLE
feat: customize HTTP/HTTPS proxy URL

### LINKED ISSUE #18: Allow to specifiy proxy URL for HTTP and HTTPS
Hello,

Some proxy providers have a different port for HTTP and HTTPS connections

https://github.com/gajus/puppeteer-proxy/blob/f73ecf0eaaef9f18bcd5768739c683a202146e31/src/routines/proxyRequest.js#L123

what if the library exposes a way to provide both things?

Something like that:

```js
agent = {
  http: new HttpProxyAgent(proxyUrl.http || proxyUrl),
  https: new HttpsProxyAgent(proxyUrl.https || proxyUrl)
};
```

### PR DESCRIPTION
**Pull request by Kikobeats**
closes #18

### CODE REVIEW

**Comment by Kikobeats** [comment]
now green @gajus 👌

File: test/puppeteer-proxy/routines/proxyRequest.js [hunk 1]
```diff
@@ -66,6 +66,41 @@ test('proxies a GET request', async (t) => {
   t.true(requestHandler.called);
 });
 
+test('specify HTTP and HTTP proxy', async (t) => {
```
Comment (gajus) [inline test/puppeteer-proxy/routines/proxyRequest.js]: Should say _and HTTPS_

**Comment by gajus** [comment]
Thanks.

Just need to update `README.md` to match the new API.

File: test/puppeteer-proxy/routines/proxyRequest.js (hunk 1 above)
Comment (Kikobeats) [reply test/puppeteer-proxy/routines/proxyRequest.js]: ops, fixed

**Comment by Kikobeats** [comment]
done!

**Comment by gajus** [comment]
Thank you

**Pull request merged**